src/player.cpp

```diff
@@ -11302,10 +11299,6 @@ void player::do_read( item *book )
         add_msg(m_info, _("You can no longer learn from %s."), book->type_name().c_str());
     }
 
-    for( auto &m : reading->use_methods ) {
```

Comment:
Well, that makes them completely useless. You could at least move the action into the itype directly and still invoke them after reading the item.